src/index.html

```diff
@@ -1,22 +1,524 @@
+          >
+            <h2 class="section-title">Contact us</h2>
+            <div class="contacts__wrap">
+              <article
+                class="contacts__information contacts__information--first"
+              >
+                <h3 class="contacts__subtitle">The Met Fifth Avenue</h3>
+                <p class="contacts__text">Phone</p>
+                <a
+                  href="tel:+1 212-535-7710"
+                  class="contacts__call-number"
+                >
+                  +1 212-535-7710
+                </a>
+                <p class="contacts__text">Address</p>
+                <a
+                  class="contacts__address"
+                  href="http://maps.google.com/?q=1000 Fifth Avenue New York, NY 10028"
+                >
+                  1000 Fifth Avenue New York, NY 10028
+                </a>
+              </article>
+              <article
+                class="contacts__information contacts__information--second"
+              >
+                <h3 class="contacts__subtitle">The Met Cloisters</h3>
+                <p class="contacts__text">Phone</p>
+                <a
+                  href="tel:+1 212-923-3700"
+                  class="contacts__call-number"
+                >
+                  +1 212-923-3700
+                </a>
+                <p class="contacts__text">Address</p>
+                <a
+                  class="contacts__address"
+                  href="http://maps.google.com/?q=99 Margaret Corbin Drive Fort Tryon Park New York, NY 10040"
+                >
+                  99 Margaret Corbin Drive Fort Tryon Park New York, NY 10040
+                </a>
+              </article>
+              <form
+                onsubmit="this.reset(); return false;"
+                class="contacts__form"
+                method="get"
+              >
+                <div class="contact__field">
+                  <input
+                    type="text"
+                    class="contact__field contact__field--short"
+                    id="name"
+                    name="name"
+                    placeholder="Name"
+                    required
+                  />
+                  <input
+                    type="email"
+                    class="contact__field contact__field--short"
+                    id="email"
+                    size="30"
+                    placeholder="Email"
+                    required
+                  />
+                  <textarea
+                    class="contact__field contact__field--long"
+                    id="message"
+                    placeholder="Message"
+                    required
+                  ></textarea>
+                </div>
+                <input
+                  class="contact__button"
+                  type="submit"
+                  value="Send"
+                />
+              </form>
+            </div>
+          </section>
+        </div>
+      </div>
+    </main>
+
```

Comment:
Add empty lines between multiline sibling blocks of HTML

![image](https://github.com/user-attachments/assets/c7537c2a-ae46-4717-8f04-0cc9c0c6185d)